lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -384,40 +385,26 @@ function isRelevantPr(
 } /* v8 ignore stop */
 
 // TODO: coverage (#9624)
-/* v8 ignore start */
-export async function getPrList(refreshCache?: boolean): Promise<Pr[]> {
+export async function getPrList(): Promise<Pr[]> {
   logger.debug(`getPrList()`);
-  if (!config.prList || refreshCache) {
-    const searchParams: Record<string, string> = {
-      state: 'ALL',
-    };
-    if (!config.ignorePrAuthor && config.username !== undefined) {
-      searchParams['role.1'] = 'AUTHOR';
-      searchParams['username.1'] = config.username;
-    }
-    const query = getQueryString(searchParams);
-    const values = (
-      await bitbucketServerHttp.getJsonUnchecked<BbsRestPr[]>(
-        `./rest/api/1.0/projects/${config.projectKey}/repos/${config.repositorySlug}/pull-requests?${query}`,
-        { paginate: true },
-      )
-    ).body;
+  const prs = await BbsPrCache.getPrs(
+    bitbucketServerHttp,
+    config.projectKey,
+    config.repositorySlug,
+    config.ignorePrAuthor,
+    config.username,
+  );
+  logger.debug(`Retrieved Pull Requests, count: ${prs.length}`);
```

Comment:
we don't log this in other platforms, so we shouldn't here too or do on all platforms